src/index.html

```diff
@@ -13,6 +13,66 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
```

Comment:
Add empty lines between multiline sibling blocks of HTML to simplify reading. There should be an empty line before this `div` element.